config/initializers/secure_headers.rb

```diff
@@ -2,7 +2,10 @@
 
 if FeatureManagement.rails_csp_tooling_enabled?
   Rails.application.configure do
-    config.ssl_options = { hsts: { preload: true, expires: 1.year, subdomains: true } }
+    config.ssl_options = {
+      secure_cookies: true,
```

Comment:
it looks like this will make rails set the `secure` option in cookie headers, but i'm not seeing how it sets `samesite` or `httponly`?

https://github.com/rails/rails/blob/f95c0b7e96eb36bc3efc0c5beffbb9e84ea664e4/actionpack/lib/action_dispatch/middleware/ssl.rb#L114-L126

Reply:
I think we may need to add `SameSite` and `HttpOnly` to the end of cookies https://github.com/github/secure_headers/blob/ce2ad139646f9775061159fe5c4707638fbe45c1/lib/secure_headers/headers/cookie.rb#L40-L45

We can add specs to catch this too

Reply:
It looks like Rails 6 gives us samesite as lax by default https://github.com/rails/rails/commit/cd1aeda0a9dc15f09d7bf1b8b59e2ce07946f031
I verified by checking `Rails.application.config.action_dispatch.cookies_same_site_protection` in the rails console.

Reply:
checking the cookies on firefox dev tools it looks like rails sets it to httponly, but i'm still trying to find where they set it explicitly.

Reply:
I just checked out this branch locally and ran it with HTTPS `HOST=0.0.0.0 make run-https`

 
```
identity-idp:abradley/lg-5772-secure-cookies> curl -k -v https://localhost:3000 2>&1 | grep -i cookie
< Set-Cookie: ahoy_visitor=e4eb7961-0420-46ce-b2ed-85f87a86c947; path=/; expires=Sun, 18 Feb 2024 18:40:16 GMT; SameSite=Lax
< Set-Cookie: ahoy_visit=c2ba3bd7-11b3-4914-90d1-102180268a43; path=/; expires=Fri, 18 Feb 2022 18:55:16 GMT; SameSite=Lax
< Set-Cookie: ahoy_track=true; path=/; SameSite=Lax
< Set-Cookie: _upaya_session=0d8395ea6299e025ca32542c769b0bf3; path=/; HttpOnly; SameSite=Lax
```

Compared to production
- All the cookies are missing `secure`
- All the "ahoy" cookies are missing `httponly`

I think we're going to need to make a custom middleware that adds these to the all the cookies, because the ahoy ones are set by a separate gem so I think the middleware layer is the right place to intercept that